xbmc/cores/dvdplayer/DVDDemuxers/DVDDemuxFFmpeg.cpp

```diff
@@ -439,6 +439,19 @@ bool CDVDDemuxFFmpeg::Open(CDVDInputStream* pInput, bool streaminfo, bool filein
 
   if (m_streaminfo)
   {
+    if (CSettings::GetInstance().GetBool(CSettings::SETTING_AUDIOOUTPUT_SUPPORTSDTSHDCPUDECODING))
```

Comment:
> EDIT: and also users with passthrough want to see their audio tracks labelled DTS HD MasterAudio

This is your opinion, yes. Mine is / was different.

And concerning what I said: No - I am not wrong, what I said was and is still perfectly right, if we want it that way is a complete different topic - guess what - I don't care at all, it can be done like the one ontop of the roof or like it is under the table of the hemple family ... 

Btw. the setting was introduced only, cause others asked for it (low performance platform whatever), see the history of this PR. It is only there, cause I wanted to speed up integration and went for the compromise ...